client/post-editor/editor-tags/index.jsx

```diff
@@ -63,28 +63,27 @@ export default React.createClass( {
 	},
 
 	render: function() {
-		var tagNames = ( this.props.tags || [] ).map( function( tag ) {
+		const tagNames = ( this.props.tags || [] ).map( function( tag ) {
 			return tag.name;
 		} );
 
+		const helpText = isPage( this.props.post )
+			? this.translate( 'Use tags to associate more specific keywords with your pages.' )
+			: this.translate( 'Use tags to associate more specific keywords with your posts.' );
+
 		return (
-			<label className="editor-drawer__label">
-				<span className="editor-drawer__label-text">
+			<div>
```

Comment:
Does this change the effect of clicking the label? Unsure if it works with `<TokenField />` but default behavior of clicking a label which has an `input` child is to focus that child, which seems like it would be broken here. Not a huge issue I suppose.